Explorer/Assets/Scripts/Global/Dynamic/MainSceneLoader.cs

```diff
@@ -137,10 +137,15 @@ private async UniTask InitializeFlowAsync(CancellationToken ct)
                     return;
                 }
 
+
                 Entity playerEntity;
                 (globalWorld, playerEntity) = bootstrap.CreateGlobalWorldAndPlayer(bootstrapContainer, staticContainer!, dynamicWorldContainer!, debugUiRoot);
                 await bootstrap.LoadStartingRealmAsync(dynamicWorldContainer!, ct);
                 await bootstrap.UserInitializationAsync(dynamicWorldContainer!, globalWorld, playerEntity, splashScreenAnimation, splashRoot, ct);
+
+                //TODO: Implement loading this (or more addresses) from Unleash
+                var pxEns = "globalpx.dcl.eth";
```

Comment:
We should delete this, instead add a configurable field in `DebugSettings` with the desired px?

Reply:
Sounds good! at least until we configure the Unleash link